kompose.spec

```diff
@@ -23,12 +12,15 @@
 %global with_unit_test 0
 %endif
```

Comment:
> so what I am proposing to do as part of this rpm build is to build kompose with bundled deps under the vendor folder (as they do upstream). Which one of these macros do I need to set?

just ``with_bundled 1``, the else branch can stay as it is unless you want to build the package on CentOS and RHEL7 as well. Then set ``with_bundled 1`` in both.

> I notice openshift origin package builds from bundled deps and does not have a devel package. Is there a reason for that?

The devel subpackage is optional, yet recommended. The main reason was most likely the time spent debundling as the devel package itself without its dependencies is not usable.